src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/config.jelly

```diff
@@ -20,6 +20,14 @@
   <f:advanced>
     <f:entry title="${%Plan Labels}" field="labels">
       <f:select multiple="multiple" size="5" />
+      <f:prepareDatabinding/>
+      ${descriptor.calcFillSettings(field,attrs)}
```

Comment:
for my education, where is the field coming from?

Reply:
Kim - the prepareDatabinding tag will set that. attrs is also built-in.